Filecontent_indexhtml and css/index.html

```diff
@@ -0,0 +1,275 @@
+		<div class="work_contains">
+	   <img class="add_snaphoot1" src="./Images/Snapshoot2.png">
+
+	   <div class="info_work_cont">
+		 <h1 class="Add align-item">Multi-Post<h1><h1>stories</h1>
+	   </div>
+
+	   <div class="description_a_items ">
+		 <p class="company"> CANOPY </p>
+		 <div class="separator"></div>
+
+		 <p class="responsability"> Back End Dev </p>
+
+		 <div class="separator"></div>
+		 <p class="year"> 2015 </p>
+	   </div>
+
+	   <div class="projects_descriptions">
+		 <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+	   </div>
+
+	  
+	   <div class="programming_icon">
+
+		<a href="#" target="_blank" id="html1"><img src="Images/html.png"> </a>
+
+		<a href="#" target="_blank" id="html1"><img src="Images/css.png"> </a>
+		
+		<a href="#" target="_blank" id="html1"><img src="Images/javascript.png"> </a>  
+	  </div>
+
+	</div>
+	   <button class="vizualization_projects"> See Project </button>
+	 </div>
+	 </section>
+
+	  <section class="contains_work_section">
+		<div class="work_contains">
+	   <img class="add_snaphoot1" src="./Images/Snapshoot3.png">
+
+	   <div class="info_work_cont">
+		 <h1 class="Add align-item">Tonic</h1>
+	   </div>
+
+	   <div class="description_a_items ">
+		 <p class="company"> CANOPY </p>
+		 <div class="separator"></div>
+
+		 <p class="responsability"> Back End Dev </p>
+
+		 <div class="separator"></div>
+		 <p class="year"> 2015 </p>
+	   </div>
+
+	   <div class="projects_descriptions">
+		 <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+	   </div>
+        
+	   <div class="programming_icon">
+
+		<a href="#" target="_blank" id="html1"><img src="Images/html.png"> </a>
+
+		<a href="#" target="_blank" id="html1"><img src="Images/css.png"> </a>
+		
+		<a href="#" target="_blank" id="html1"><img src="Images/javascript.png"> </a>  
+	  </div>
+	  
+	   </div>
+	   <button class="vizualization_projects"> See Project </button>
+	 </div>
+	 </section>
+
+	 
+	 <section class="contains_work_section">
+		<div class="work_contains">
+	   <img class="add_snaphoot1" src="./Images/Snapshoot4.png">
+
+	   <div class="info_work_cont">
+		 <h1 class="Add align-item">Multi-Post</h1><h1>stories</h1>
+	   </div>
+
+	   <div class="description_a_items ">
+		 <p class="company"> CANOPY </p>
+		 <div class="separator"></div>
+
+		 <p class="responsability"> Back End Dev </p>
+
+		 <div class="separator"></div>
+		 <p class="year"> 2015 </p>
+	   </div>
+
+	   <div class="projects_descriptions">
+		 <p> A daily selection of privately personalized reads; no accounts or sign-ups required. </p>
+	   </div>
+
+	   <div class="programming_icon">
+
+		<a href="#" target="_blank" id="html1"><img src="Images/html.png"> </a>
+
+		<a href="#" target="_blank" id="html1"><img src="Images/css.png"> </a>
+		
+		<a href="#" target="_blank" id="html1"><img src="Images/javascript.png"> </a>  
+	  </div>
```

Comment:
As you have done for the media links, kindly consider using ul and li for listing the project technology list of `"JavasScript", "HTML" and "CSS"` since that is the best practice and good for semantic reasons :wink: